modules/gapi/include/opencv2/gapi/own/mat.hpp

```diff
@@ -239,6 +242,7 @@ namespace cv { namespace gapi { namespace own {
                 tmp.data = tmp.memory.get();
 
                 *this = std::move(tmp);
+                this->size = MatSize(&this->rows, &this->dims);
```

Comment:
Looks like there is something incorrect with MatHeader/MatSize operator= if it doesn't work without this line